mapie/multi_label_classification.py

```diff
@@ -138,12 +146,16 @@ def __init__(
         estimator: Optional[ClassifierMixin] = None,
         n_jobs: Optional[int] = None,
         random_state: Optional[Union[int, np.random.RandomState]] = None,
-        verbose: int = 0
+        verbose: int = 0,
+        metric_control: Optional[str] = 'recall',
+        method: Optional[str] = None
     ) -> None:
         self.estimator = estimator
         self.n_jobs = n_jobs
         self.random_state = random_state
         self.verbose = verbose
+        self.metric_control = metric_control
+        self.method = method
```

Comment:
UPD: they should be defined just after estimator (l153) and before n_jobs (l154).

Reply:
Done